testdata.sql

```diff
@@ -0,0 +1,28 @@
+
+/* Org with id 100 */
+DELETE FROM orgs_org where id = 100;
```

Comment:
Seems odd having explicit deletes in a test data file - shouldn't we just truncate these tables?